sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTest.java

```diff
@@ -1412,8 +1413,12 @@ public void checkRecognizeFormTypeComposedModel(
                 syncPoller3.waitForCompletion();
 
                 final RecognizedForm recognizedForm = syncPoller3.getFinalResult().stream().findFirst().get();
-                // since none of the models have a name
-                assertEquals(recognizedForm.getFormType(), "custom:");
+                if (recognizedForm.getFormType().equals("custom:" + createdModel1.getModelId())
```

Comment:
.getFormType() could be null.

Reply:
also, 
String formType = recognizeForm.getFormType();

assertTrue(("custom:"+createdModel1.getModelId()).equals(formType) || ("custom:"+createdModel.getModelId()).equals(formType))

Reply:
Actually, that's a service returned property so if it is returned null and the tests fail, it will be good to know.